samples/UsePolicyBuilder/Startup.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Cors.Core;
+using Microsoft.AspNet.Http;
+using Microsoft.Framework.DependencyInjection;
+
+namespace UsePolicy
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddCors();
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            app.UseCors(builder => builder.AddOrigins("http://example.com"));
```

Comment:
`policy => policy.xxx` instead of builder would be clearer. (I know, it's just sample :) )